bemani/backend/jubeat/base.py

```diff
@@ -223,13 +242,21 @@ def update_score(
             # Update the ghost regardless, but don't bother with it in history
             scoredata.replace_int_array('ghost', len(ghost), ghost)
 
+        if music_rate != 0:
```

Comment:
Prefer None instead of 0 for "value is not present".

Reply:
I'm not super skilled with python, is None a valid value for an integer? If so I'll just change this and the default value for this to None